<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>KST_EXPORT</exportmacro>
 <class>PluginDialogWidget</class>
 <widget class="QWidget" name="PluginDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>109</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox9_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Plugin Selection</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="_pluginManager">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Manager...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="PluginCombo"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="_pluginInfoFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="_pluginInputsBox">
     <property name="title">
      <string>Inputs and Outputs</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="_pluginInputOutputFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="CurveAppearanceWidget" name="_curveAppearance" native="true">
     <property name="hidden" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>CurveAppearanceWidget</class>
   <extends>QWidget</extends>
   <header>curveappearancewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>PluginCombo</tabstop>
  <tabstop>_pluginManager</tabstop>
  <tabstop>_curveAppearance</tabstop>
 </tabstops>
 <includes>
  <include location="global">kst_export.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
